File: twoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int left = 0, right = numbers.size()-1;
        vector<int> answer;
        for(int i = 0 ; i<numbers.size(); i++){
            if(numbers[left]+numbers[right]== target){
                answer.push_back(left+1);
                answer.push_back(right+1);
                break;
            }
            else if(numbers[left]+numbers[right] > target){
                right--;
            }
            else {
                left++;
            }
        }
        return answer;
    }
};

File: twoSum.py
def twoSum(nums, target):
    list = []
    for i in range(len(nums)):
        for j in range(i + 1, len(nums)):
            if nums[i] + nums[j] == target:
                list.append(i)
                list.append(j)
                break
    return list